### ISSUE TITLE
Empty npz file in geo_prior

### ISSUE THREAD

**Issue by Jiang0903** [Gardlin/PEAL#3]
Hello,

It's really a nice work !

I have some questions about the details of how you process the files in `geo_prior` as there are several empty files in `geo_prior`, like `geo_prior_tesval/7-scenes-redkitchen/5_9.npz` with 0KB size.   Are these files empty because `ref_corr_indices` and `src_corr_indices` are none under the `estimated_transform` by GeoTransformer ?

Besides, when I tried to run the pretrained model on test dataset, I met error at `prior_data = np.load(file_name)`  which locates at `PEAL/geotransformer/datasets/registration/threedmatch/dataset.py   line 214`. The error is reported because `numpy` can not load an empty `.npz` file when testing the case of `5_9 in 7-scenes-redkitchen`. How can I solve this problem?

Thanks,
Xinqi.

**Comment by Gardlin** [comment]
Hi, Xinqi,

Sorry for the empty files,   I have uploaded a new file and updated the link in the readme. The "geo_prior_testval" file contains the estimated poses and correspondences of the pretrained Geotransformer. If you'd like, you can also dump these priors for testing or validating purposes.

**Comment by ZhiChen902** [comment]
Hello, 
Thanks for your nice work and for updating the prior file for the test set. I still find there are some empty files in the priors for training set, like `geo_prior_training/bundlefusion-copyroom_1/37_9.npz` and `geo_prior_training/bundlefusion-copyroom_1/36_5.npz`. 
![1688128759476](https://github.com/Gardlin/PEAL/assets/54473539/fb3fdfc2-5b0b-444e-9b47-6d10f56b3f79)

Thanks

**Comment by ZhiChen902** [comment]
Hello,
Thanks for your nice work and for updating the prior file for the test set. I still find there are some empty files in the priors for training set, like `geo_prior_training/bundlefusion-copyroom_1/37_9.npz` and `geo_prior_training/bundlefusion-copyroom_1/36_5.npz`.
![1688129139192](https://github.com/Gardlin/PEAL/assets/54473539/1a82ea62-389a-4f02-b8ba-d4e63989771f)

Thanks

**Comment by Gardlin** [comment]
> 

Hi,  

Sorry for the empty files again,  the new training data is now available. If you still come across any remaining empty files in the training data, you can use the ground truth transformation to replace these empty "prior" values. Based on my experience, this substitution does not significantly impact the final registration performance.

**Comment by Gardlin** [comment]
> Hello, Thanks for your nice work and for updating the prior file for the test set. I still find there are some empty files in the priors for training set, like `geo_prior_training/bundlefusion-copyroom_1/37_9.npz` and `geo_prior_training/bundlefusion-copyroom_1/36_5.npz`. ![1688129139192](https://user-images.githubusercontent.com/54473539/250091530-1a82ea62-389a-4f02-b8ba-d4e63989771f.png)
> 
> Thanks

Hi,

If you still have any trouble replicating the results reported in the paper,  please let me know. And I will close this issue in two days.
